joint_trajectory_controller/test/test_trajectory_controller.cpp

```diff
@@ -98,19 +98,15 @@ class TestTrajectoryController : public ::testing::Test
     traj_msg.points.resize(points.size());
 
     builtin_interfaces::msg::Duration duration_msg;
-    duration_msg.sec = time_from_start.sec;
-    duration_msg.nanosec = time_from_start.nanosec;
+    duration_msg.sec = delay_btwn_points.sec;
+    duration_msg.nanosec = delay_btwn_points.nanosec;
     rclcpp::Duration duration(duration_msg);
     rclcpp::Duration duration_total(duration_msg);
 
     size_t index = 0;
```

Comment:
I don't remember why it's outside the for loop but I think this can be incorporated into the `for` loop

Reply:
iirc it was to fix a bug 854b1318a9d742150fcf17c3fe3efd26ceff9f96

Reply:
let's put it back in the for loop plesae, `index` not use outside. If this causes issues elsewhere, we call the ghostbusters

Reply:
put index back in loop 7851790